src/emm/services/modules_service.py

```diff
@@ -231,3 +231,45 @@ def git_commit_modules(self, commit: str) -> None:
             module_data = self.get_module_data(module)
             module_location = Path(module_data.prefix) / module
             self.git_service.commit_all(commit, module_location)
+
+    def add_pr_link(self, comments: Dict[str, str]) -> None:
```

Comment:
[nit] This should probably be named `add_pr_links` since it is add more than 1 link.